invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -28,22 +28,20 @@ const RecordVersionItem = ({ item, activeVersion }) => {
     <>
       <Grid.Row
         key={item.id}
-        columns={2}
+        columns={1}
         {...(activeVersion && { className: "version-active" })}
       >
-        <Grid.Column width={10}>
+        <Grid.Column>
+        <small className="text-muted" style={{ float: "right" }}>
+             {item.publication_date}
+           </small>
           <a href={`/records/${item.id}`}>Version {item.version}</a>
-          {doi && (
-            <small className="text-muted" style={{ float: "left" }}>
-              {doi}
+          {<br/>}{doi && (
+            <small  className="text-muted">
+              {doi } 
```

Comment:
minor: extra space
```suggestion
              {doi} 
```

Reply:
Thanks Pablo :D 
I squashed the commits and deleted that extra space
I will upload 2 screenshots of the window in a different size, the doi will break into a new line so it doesnt overflow

![versionbreak2](https://user-images.githubusercontent.com/30771373/116991036-42e0b900-acd4-11eb-99af-880fd64b7303.png)

![versionbreak](https://user-images.githubusercontent.com/30771373/116990806-ebdae400-acd3-11eb-9342-7ceb6b6782d2.png)